docs/telemetry.md

```diff
@@ -1,99 +1,157 @@
+
+- `ENABLE_OPEN_TELEMETRY`: Supports `true` or `false`.  After setting this to `true` and restarting the deployment with `./release up -d` the telemetry services will be started and the Graphistry services will export telemetry data.
+- `OTEL_COMPOSE_FILE`: Indicates which telemetry services will be deployed along the Graphistry services.  Possible values are `telemetry.cloud.yml` (e.g. when we want to export to Grafana) or `telemetry.yml` (e.g. when we want to use the self-hosted Prometheus, Jaeger and Grafana instances).
+
+These environment variables are used when we want to use `OTEL_COMPOSE_FILE=telemetry.yml`:
+- `OTEL_COLLECTOR_OTLP_HTTP_ENDPOINT`: For example the Grafana Cloud OTLP HTTP endpoint.
+- `OTEL_COLLECTOR_OTLP_USERNAME`: For example the Grafana Cloud Instance ID for OTLP
+- `OTEL_COLLECTOR_OTLP_PASSWORD`: For example the Grafana Cloud API Token for OTLP
 
-To configure collection (but not yet turn on), enable the custom environment variable ENABLE_OPEN_TELEMETRY in your configuration file: `$GRAPHISTRY_HOME/data/config/telemetry.env`
+ ### Telemetry services
+
+The `$GRAPHISTRY_HOME/etc` has the directories to configure the telemetry services:
+- `$GRAPHISTRY_HOME/etc/otel-collector`: Has the `otel-collector-config.cloud.yml` (when we use the `OTEL_COMPOSE_FILE=telemetry.cloud.yml`) and `otel-collector-config.yml` (`OTEL_COMPOSE_FILE=telemetry.yml`).
+- `$GRAPHISTRY_HOME/etc/prometheus`: Has `prometheus.yml` to configure the self-hosted Prometheus service.  By default, the self-hosted Prometheus service is already configured to collect metrics from Graphistry services and from the `DCGM exporter` service.
+- `$GRAPHISTRY_HOME/etc/jaeger`: To configure the self-hosted Jaeger service.
+- `$GRAPHISTRY_HOME/etc/grafana`: To configure the self-hosted grafana service.  By default, the self-hosted Grafana service has already configured the self-hosted Prometheus service as default datasource.  Also, it has already configured and ready to use the `DCGM Exporter Dashboards`.
+
+## Examples
 
 ### Example: Graphana Cloud
-For example, this is the configuration template for [Grafana Cloud](https://grafana.com/), which includes a free tier:
+
+This is the configuration template for [Grafana Cloud](https://grafana.com/):
 
 ```bash
 ENABLE_OPEN_TELEMETRY=true
 
+OTEL_COMPOSE_FILE=telemetry.cloud.yml
+
+OTEL_COLLECTOR_OTLP_HTTP_ENDPOINT="https://hostname.grafana.net/otlp"   # e.g. Grafana OTLP HTTP endpoint
 OTEL_COLLECTOR_OTLP_USERNAME="XYZ"   # e.g. Grafana Cloud Instance ID for OTLP
 OTEL_COLLECTOR_OTLP_PASSWORD="PQR"   # e.g. Grafana Cloud API Token for OTLP
-OTEL_COLLECTOR_OTLP_HTTP_ENDPOINT="https://hostname.grafana.net/otlp"   # e.g. Grafana OTLP HTTP endpoint 
-OTEL_COLLECTOR_COMPOSE_FILE=otel-collector.yml
-OTEL_COLLECTOR_CONF_FILE=./etc/otel-collector/otel-collector-config.yml
 ```
 
-### Example: Local Jaeger and Prometheus
+### Example: Self-hosted Prometheus, Jaeger and Grafana instances
```

Comment:
I think self-hosted might not be the best term here, perhaps adding a definition of what is meant by self-hosted, because they could also be self-hosting the services elsewhere and this might add confusion.

Reply:
Thanks, yes I've changed from `self-hosted` to `Graphistry` Local Telemetry Suite` and added this brief definition in the Overview section:
`Graphistry can be deployed alongside the packaged Graphistry Local Telemetry Suite, which includes Prometheus, Jaeger, and Grafana services.`